uwblueprint/extend-a-family
# Aathi's onboarding

## Notion ticket link

[Onboarding](https://www.notion.so/uwblueprintexecs/Setup-Onboarding-Task-d6e0cdeb69cf4bbf93a33c7302370550)



## Implementation description
* Query and Create User implementation



## Steps to test
1. Go to Team Members Page
2. Try to create a User by pressing the "Add Team Member" button
3. Table should be updated, MongoDB Cluster should be updated



## What should reviewers focus on?
* 


## Checklist
